File: LeetCode/1848. Minimum Distance to the Target Element/solution.py
class Solution:
    def getMinDistance(self, nums: List[int], target: int, start: int) -> int:
        n = len(nums)
        res = n 
        for i in range(n):
            if nums[i] == target:
                res = min(res, abs(i-start))
        return res

File: LeetCode/1848. Minimum Distance to the Target Element/solution.cpp
class Solution {
public:
int getMinDistance(vector<int>& nums, int target, int start) {
    int res = INT_MAX;
    for (int i = 0; i < nums.size() && res > abs(start - i); ++i)
        if (nums[i] == target)
            res = abs(start - i);
    return res;
}
};